src/Mod/Sketcher/App/SketchObject.h

```diff
@@ -476,6 +476,11 @@ class SketcherExport SketchObject: public Part::Part2DObject
      * \return -1 on error
      */
     int exposeInternalGeometry(int GeoId);
+    template<class geomType>

```

Comment:
```suggestion
    template<class GeomType>
```

Let's stick with `PascalCase` names for type placeholders